lib/taiko.js

```diff
@@ -198,6 +198,11 @@ module.exports.openBrowser = async (options = { headless: true }) => {
       'Invalid option parameter. Refer https://taiko.gauge.org/#parameters for the correct format.',
     );
   }
+
+  if (chromeProcess && !chromeProcess.killed) {
+    throw new Error('openBrowser cannot be called again as there is a chromium instance open (closeBrowser is not yet called)');
```

Comment:
Can you change the message to `OpenBrowser cannot be called again as there is a chromium instance open.';

Reply:
@Apoorva-GA Sure will make the changes